main.yml

```diff
@@ -644,6 +646,23 @@ openshift:
 
   top_level: true
 
+ocp-advisor: 
+  title: Advisor
+  source_repo: https://github.com/RedHatInsights/ocp-advisor-frontend
+  deployment_repo: https://github.com/RedHatInsights/ocp-advisor-frontend-build
+  frontend:
+    module: ocp-advisor#./RootApp
+    isHidden: true
```

Comment:
Just a question to make sure: when `isHidden` is set to true, then the app is not available in all envs within its path, right?

Reply:
@gkarat ishidden ensures the entry does not show up in the lnav this pr only adds a csc record for ci-beta